### PR TITLE
Kernel_23: inconsistent documentation

### PR DESCRIPTION
**Pull request by albert-github**
Made preconditions in the C++ form.

### CODE REVIEW

File: Kernel_23/doc/Kernel_23/CGAL/Direction_2.h [hunk 1]
```diff
@@ -60,7 +60,7 @@ Direction_2(const Kernel::RT &x, const Kernel::RT &y);
 
 /*!
 returns values, such that `d``== Direction_2<Kernel>(delta(0),delta(1))`.
-\pre \f$ 0 \leq i \leq1\f$.
+\pre `0 <= i <= 1`.
```
Comment (lrineau) [inline Kernel_23/doc/Kernel_23/CGAL/Direction_2.h:63]: Do we want to have incorrect C++ expressions, in preconditions? The alternative could be:
```suggestion
\pre `0 <= i` and `i <= 1`.
```

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.6-Ic-84

**Comment by lrineau** [comment]
This pull-request seems still under discussion. @MaelRL, can you tell us if the discussion is resolved or not?

**Comment by albert-github** [comment]
@lrineau in the comment https://github.com/CGAL/cgal/pull/6916#discussion_r984595658 I did some suggestions about alternative solutions, of which I think only the first one is really in the C++ style (see also #6891), but there has been no further discussion so far.
In my opinion the chosen version (`` \pre `0 <= i <= 1` ``) is OK , alternatively it would be possible to change this type of construct on a number of places into `` i == 0 || i == 1 `` (to keep the idea of the C++ style as mentioned in #6891).

**Comment by afabri** [comment]
I think that `0 <= i <=1`  is ok in the documentation.  For just two possible values I would prefer "`i == 0`  or `i == 1`".  But all that does not really matter, as what is important is that we do not use `f$`.

**Pull request merged**